salt/reactors/opsgenie/post_alert.sls

```diff
@@ -0,0 +1,9 @@
+post_alert_to_opsgenie:
+  local.opsgenie.post_data:
+    - tgt: 'roles:master'
+    - tgt_type: grain
+    - kwarg:
+        name: salt.reactor.alert
+        api_key: __vault__::secret-operations/global/opsgenie/opsgenie_saltstack_api>data>value
+        reason: {{ data['data']['reason'] }}
```

Comment:
Is it possible to send the entire data object as an additional field?

Reply:
Don't believe so. Think the only think we can maybe do is send it in the `reason` key. The salt stack integration appears to be fairly limited - https://docs.opsgenie.com/docs/saltstack-integration